Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -265,6 +267,14 @@ protected function createRoutesFromConfiguration()
                 }
                 $routesWithHttpMethodConstraints[$uriPattern] = false;
             }
+            if (isset($routeConfiguration['cache'])) {
+                if (isset($routeConfiguration['cache']['lifetime']) && is_int($routeConfiguration['cache']['lifetime'])) {
```

Comment:
slightly more defensive would be something along the lines of
```suggestion
                if (isset($routeConfiguration['cache']['lifetime'])) {
                    assert(is_int($routeConfiguration['cache']['lifetime']), 'cache.lifetime must be of typ e integer');
```

Reply:
see below